independent-publisher-connectors/SOS Inventory/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2231 @@
+												"example": "0"
+											},
+											"useMarkup": {
+												"type": "boolean"
+											},
+											"minimumPrice": {
+												"type": "number"
+											},
+											"basePurchaseCost": {
+												"type": "number",
+												"example": "11.66"
+											},
+											"purchaseCost": {
+												"type": "number",
+												"example": "8.0130144145"
+											},
+											"costBasis": {
+												"type": "number",
+												"example": "-92.27477"
+											},
+											"weight": {
+												"type": "number",
+												"example": "0"
+											},
+											"volume": {
+												"type": "number",
+												"example": "0"
+											},
+											"weightUnit": {
+												"type": "string",
+												"example": "lb"
+											},
+											"volumeUnit": {
+												"type": "string",
+												"example": "cbm"
+											},
+											"sublevel": {
+												"type": "integer",
+												"example": "0"
+											},
+											"taxable": {
+												"type": "boolean"
+											},
+											"salesTaxCode": {
+												"type": "string"
+											},
+											"purchaseTaxCode": {
+												"type": "string"
+											},
+											"willSync": {
+												"type": "boolean"
+											},
+											"updateShopify": {
+												"type": "boolean"
+											},
+											"updateBigCommerce": {
+												"type": "boolean"
+											},
+											"alwaysShippable": {
+												"type": "boolean"
+											},
+											"hasImage": {
+												"type": "boolean"
+											},
+											"serialTracking": {
+												"type": "boolean"
+											},
+											"lotTracking": {
+												"type": "boolean"
+											},
+											"archived": {
+												"type": "boolean"
+											},
+											"showOnSalesForms": {
+												"type": "boolean"
+											},
+											"showOnPurchasingForms": {
+												"type": "boolean"
+											},
+											"showOnManufacturingForms": {
+												"type": "boolean"
+											},
+											"customFields": {
+												"type": "string"
+											},
+											"uoms": {
+												"type": "array",
+												"items": {}
+											},
+											"priceTiers": {
+												"type": "array",
+												"items": {}
+											},
+											"summaryOnly": {
+												"type": "boolean"
+											},
+											"imageAsBase64String": {
+												"type": "string"
+											},
+											"imageChanged": {
+												"type": "boolean"
+											},
+											"commissionRate": {
+												"type": "number",
+												"example": "0"
+											},
+											"commissionAmount": {
+												"type": "number",
+												"example": "0"
+											},
+											"commissionExempt": {
+												"type": "boolean"
+											},
+											"syncMessage": {
+												"type": "string",
+												"example": ""
```

Comment:
@Harold-Anderson please remove empty example properties `"example": ""`. There are a few of them in the file.